src/components/Books.js

```diff
@@ -1,23 +1,49 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Book from './book/Book';
 import AddBook from './AddBook';
+import './App.css';
+import { setBooks } from '../redux/books/books';
+import BookStoreService from '../api/apiServices';
 
 function Books() {
   const bookList = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  const reformulateData = (data) => {
+    const books = Object.entries(data);
+    return books.map((element) => ({ item_id: element[0], ...element[1][0] }));
+  };
+  useEffect(() => {
+    const fetchBooks = async () => {
+      try {
+        const response = await BookStoreService.apiGetBooks();
+        console.log('from books js::', response.data);
```

Comment:
- [ ] I would recommend you remove this console log here since it is unnecessary.